packages/angular/src/schematics/init/init.ts

```diff
@@ -22,7 +22,7 @@ import {
 import { Schema } from './schema';
 import { E2eTestRunner, UnitTestRunner } from '../../utils/test-runners';
 import { stripIndents } from '@angular-devkit/core/src/utils/literals';
-import { setDefaultCollection } from '@nrwl/workspace/src/utils/rules/workspace';
+import { setDefaultCollection } from '@nrwl/workspace';
```

Comment:
Thanks :)

Can you please also collapse this statement with the previous `@nrwl/workspace` import statement? (line 15)

Reply:
Ha, I should have caught that. Fixed!